package.json

```diff
@@ -46,12 +46,12 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^2.0.2",
     "sinon": "^1.17.7",
-    "underscore": "^1.9.1",
+    "underscore": "1.13.0-2",
```

Comment:
Any reason why we're not doing `^1.13.0-2`?

Reply:
Im honestly unsure about that. That symbol means that it will grab that version unless a newer version is available. But then we also have the yarn.lock file which locks the versions of things to be what we way, so its a little confusing. I can keep that carrot symbol in though to remain with convention

Reply:
"The caret (aka hat) symbol, ^, is used by default when you npm install --save a package. For example, npm install --save angular would add "angular": "^1.3.15" to the dependencies in your package.json. This tells npm that you want either 1.3.15 or a version that is greater than that but still starts with 1 (it is in the same major range)."

That makes me feel like we dont want to use the caret... idk if it matters though since we use the yarn.lock file as well

Reply:
Let's maintain the caret and update `.lock` as well

`underscore@1.13.0-2` -> `underscore@^1.13.0-2`

Reply:
Update:

yarn is weird. Honestly, the carets dont make sense at all since we use a lock file. They may be leftovers from how we did things before or something or just no one has run into this issue with the ones that still have carets. The lock file says, “no matter what, this is the version we are using”. So having the caret in the package.json is ignored anyway (unless someone runs a `yarn upgrade` command). But since this is a new install of a new package version, it is paying attention to the caret and says, “Oh hey, there is a newer version than what you said, so let me get that for you and put it in the lock file”. And we dont want that right now

Reply:
i think we should use caret so that libs that use ventana aren't tied to ventana's version of underscore. yarn should resolve the different underscore versions and pick the one that works for all the libs

Reply:
updated to "^1.13.1"